packages/client/src/api/wargames_api/index.ts

```diff
@@ -725,6 +738,43 @@ export const postNewMessage = async (dbName: string, details: MessageDetails, me
   })
 }
 
+/**
```

Comment:
IMHO @lilitkarapetyan - the method description in the comment is valuable, but the two @param comments aren't necessary - they don't tell us anything we can't see for ourselves.